test/test_broker/jobs.ScheduleBackupJob.spec.js

```diff
@@ -59,6 +59,18 @@ describe('Jobs', function () {
       const transactionLogsPathname19 = `/${serviceContainer}/${transactionLogsFileName19Daysprior}`;
       const transactionLogsPathname16 = `/${serviceContainer}/${transactionLogsFileName16DaysPrior}`;
       const transactionLogsPathname18 = `/${serviceContainer}/${transactionLogsFileName18DaysPrior}`;
+      const dummyDeploymentResource = {
+        spec: {
+          options: JSON.stringify({
+            service_id: service_id,
+            plan_id: plan_id,
+            context: {
+              platform: 'cloudfoundry',
+            },
+            space_guid: space_guid,
+          })
+        }
+      };
```

Comment:
Check if this reference of cloudcontroller is required : [jobs.ScheduleBackupJob.spec.js#L791](https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/ea6f37cef12aa1de7d9b9d5c678acbdb2e165456/test/test_broker/jobs.ScheduleBackupJob.spec.js#L791)

Reply:
Not required. Had missed it as it doesn't have mocks.verify(). Will make the appropriate changes. Thanks!